src/main/resources/templates/methodNotification.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script>
+		var getUrl = window.location;
+		window.parent.postMessage("3DS.Notification.Received", location.origin);
```

Comment:
Either use the already declared variable `getUrl` (which, btw, could enjoy a better name maybe?) or use `window.location`

Reply:
Something like that? https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/202/commits/0a171400565cb336c7e28250391917adffc2f4ab